### ISSUE TITLE
License question

### ISSUE THREAD

**Issue by malcoriel** [toconnell/kdm-manager#529]
Hello, there is no license in the description or README or LICENSE file in this repo. I wonder, which one it is?

**Comment by toconnell** [comment]
That is actually kind of a tricky question for this project: I obviously do not own (or have any explicit permission to use) any of Kingdom Death's proprietary data (e.g. gear card info, etc.) and much of the application's design intentional mimics the printed materials that come with _Monster_, so they're not really my designs.

As for the application architecture and the code itself, most of it is cookie-cutter: most of what you see in the code itself is adapted from tutorials, so there isn't really any new or proprietary technology or techniques to license or protect. 

tl;dr I never added a license to the project, because what's in the project either a.) is not mine to license or b.) is not novel or licenseable. 

Why do you ask?

**Comment by malcoriel** [comment]
Well, I am not an expert in copyright law, the question is mostly about code re-use. Even if you do not invent anything new it's not exactly okay to e.g. fork and modify your code for even private purposes if there is no license.  Say, I've copied your random name assets, which are obviously not a part of any KDM copyright, would you mind it? Or made a fork, stripped it of anything even remotely related to design or any graphics of KDM, would you say that I had infringed your copyright? The fact that it is open-sourced does not really mean much.

So it's mostly not about patenting, it's rather about allowing others to freely work with your code. Perhaps you could make whole code MIT, put all game info into a separate folder, and mention in README that game assets are not licensed appropriately and not free to copy (though I really doubt that Poots would bother).

**Comment by malcoriel** [comment]
By the way, as I see people do not care much, as this repo has 7 forks. Still, it would be nice to have some clarity.

**Comment by toconnell** [comment]
@malcoriel You know, honestly? I've never really thought about this stuff and I do not know much about it, but you make some truly excellent points here.

I am kind of in the middle of a major organizational refactor right now, so right now is kind of a great time to update the README.md and the licensing and make those sorts of changes. In fact, I have mostly segregated Poots' proprietary data into a single folder and I generally don't have a problem with the MIT license for this project, so I will probably do it almost exactly the way you say.

Thanks for bringing this up! I will be making some changes soon!

**Comment by malcoriel** [comment]
You're welcome. Thanks for being so open to changes.
